app/views/filter/conditionPanel.scala.html

```diff
@@ -11,9 +11,9 @@
     @filter.map(_.conditions).getOrElse(Nil).zipWithIndex.map { case (condition, index) =>
     <div id="condition-full@index" class="filter-part condition-full">
         <span class="condition">
-            <span class="label label-primary" id="fieldLabel">@condition.fieldLabelOrElseName</span>
-            <span id="conditionType"><b>@conditionTypeToReadableOperator(condition.conditionType)</b></span>
-            <span class="label label-primary" id="value">@valueOrUndefined(condition, trim = true)</span>
+            <span class="filter-label label label-primary" id="fieldLabel">@condition.fieldLabelOrElseName</span>
```

Comment:
I would maybe keep the field label (not the condition type or value) bold or a bit larger to make it distinguishable... but first I will check how it looks like

Reply:
This looks fine as it is